CONTRIBUTING.md

```diff
@@ -1,8 +1,44 @@
 # How to Contribute
 
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
GH does this automatically, I'm fine if you want to keep this though

Reply:
Whoa this icon?!

<img width="493" alt="image" src="https://user-images.githubusercontent.com/251288/177891005-462d526d-5f88-4f25-9320-0640265138a6.png">

I figure if I didn't know about it, many/most don't. Also nice to see the whole thing at once, vs built-in is scrolled.